scripts/install_python_prod_dependencies.py

```diff
@@ -35,7 +35,6 @@
 MismatchType = Dict[str, Tuple[Optional[str], Optional[str]]]
 ValidatedMismatchType = Dict[str, Tuple[str, Optional[str]]]
 
-# This is the version that is set in install_prerequisites.sh.
```

Comment:
Not sure what this comment means, so I removed it (it doesn't seem to relate to the next line at all).